internal/build/util.go

```diff
@@ -140,6 +140,29 @@ func CopyFile(dst, src string, mode os.FileMode) {
 	}
 }
 
+// CopyFolder copies a folder.
+func CopyFolder(dst, src string, mode os.FileMode) {
```

Comment:
Probably OK to depend on `github.com/cespare/cp` for this. We already depend on it for something else.

Reply:
Switched both CopyFile and CopyFolder over. Does this look ok?